src/domain/base/EntityHash.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { describe, it, expect } from "vitest"
+import { EntityHash } from "./EntityHash"
+import { ValueObject } from "./ValueObject"
+import { Hash } from '../valueObjects/Hash'
+
+interface TestProps {
+  value: string
+}
+
+class TestEntityHash extends EntityHash<TestProps> {
+  static create(props: TestProps, hash: Hash): TestEntityHash {
+    return new TestEntityHash(props, hash)
+  }
+
+  get value(): string {
+    return this.props.value
+  }
+}
+
+const hash = Hash.create('0x7304dc174aab2bc487b1befb9e35ba3632b9693f0c0548e138b4401f263910f1')
+
+describe("src/domain/EntityHash", () => {
+  it("should be defined", () => {
+    expect(EntityHash).toBeDefined()
+  })
```

Comment:
This can never fail. Compilation of Typescript would fail otherwise (So, the compiler would warn you already If `EntityHash` somehow wasn't defined). 

I believe this kind of tests can be dropped.